dts/bindings/audio/wolfson,wm8904.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2023, NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: WM8904 audio codec
+
+include: [i2c-device.yaml, pinctrl-device.yaml]
+
+compatible: "wolfson,wm8904"
+
+properties:
+  reg:
+    required: true
+
+  clock-source:
+    type: string
+    default: "MCLK"
```

Comment:
https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#rules-for-default-values

Reply:
let me add descriptions